frost-core/src/frost/identifier.rs

```diff
@@ -25,6 +25,23 @@ where
     }
 }
 
+impl<C> Display for Identifier<C>
+where
+    C: Ciphersuite,
+{
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(
+            f,
+            "{}",
+            hex::encode(
+                <<<C as Ciphersuite>::Group as Group>::Field as Field>::little_endian_serialize(
+                    &self.0
+                )
+            )
```

Comment:
I'm a bit keen into using big endian instead because it reads more naturally as a string, but I don't want Yet Another Endianness Discussion, so I'm happy with whatever :rofl:

Reply:
Little endian! Littlest endians! 
![](http://www.simbasible.com/wp-content/uploads/2021/09/1-4.gif)

Reply:
(
<img width="447" alt="image" src="https://user-images.githubusercontent.com/552961/203425524-673afbab-61a8-44a3-8cfb-37c767f92bed.png">
)